AutomatedTesting/Gem/PythonTests/Physics/TestSuite_Main_Optimized.py

```diff
@@ -54,7 +54,7 @@ def wrap_run(cls, instance, request, workspace, editor, editor_test_results, lau
             fm._restore_file(f, file_list[f])
 
 
-@pytest.mark.xfail(reason="Optimized tests are experimental, we will enable xfail and monitor them temporarily.")
+#@pytest.mark.xfail(reason="Optimized tests are experimental, we will enable xfail and monitor them temporarily.")
```

Comment:
Remove commented out stuff. Git history on the file can be used to recover it if necessary.

Reply:
Thought I removed, will do